src/interface/src/app/services/share-map.service.ts

```diff
@@ -0,0 +1,58 @@
+import { Inject, Injectable } from '@angular/core';
+import { map, Observable } from 'rxjs';
+import { BackendConstants } from '../backend-constants';
+import { HttpClient } from '@angular/common/http';
+import { WINDOW } from './window.service';
+import { MapConfig, MapViewOptions, Region } from '../types';
+
+interface ViewState {
+  mapViewOptions: MapViewOptions | null;
+  mapConfig: MapConfig[];
```

Comment:
The nittiest of nits:
I was just curious if you intended for this to be different than what's in the Local Storage, which has `mapConfigs`, rather than `mapConfig`.

Reply:
The one in localstorage is a `Record<Region , MapConfig[]>` , this one is just `MapConfig[]`
Its not super clear, probably we need to rename the one on localstorage...but Im not sure I wanna go that deep into the rabbit hole.

Should this one be called mapConfigs either way to represent that is an array?